README.md

```diff
@@ -15,7 +72,342 @@ def deps do
+          |> cast(params, [:title, :blocks])
+          # 5. call `validate_blocks` passing the name of your field
+          |> validate_blocks(:blocks)
+        end
+      end
+      ```
+
+      **Note:** `validate_blocks/2` can take an atom or a list of atoms if you have more than one set of blocks fields to validate.
+
+  2. Then create the migration to add the field to your database
+
+      ```bash
+      mix ecto.gen.migration add_blocks_to_posts
+      ```
+
+  3. `EctoBlocks` expects the underlying field to be a `:map` which is implemented as a `jsonb` column in databases such as Postgres.
+
+      ```elixir
+      # priv/repo/migrations/<timestamp>_add_blocks_to_posts.exs
+
+      defmodule YourApp.Repo.Migrations.AddBlocksToPost do
+        use Ecto.Migration
+
+        def change do
+          alter table("posts") do
+            add :blocks, :map
+          end
+        end
+      end
+      ```
+
+  4. Run your migration
+
+      ```bash
+      mix ecto.migrate
+      ```
+  ---
+</details>
+
+<details>
+  <summary>
+    <strong>
+      4. Teach your Phoenix Views how to render blocks
+    </strong>
+  </summary>
+
+  1. Use `ColonelKurtz.Renderer.render_blocks/1` to render block content somewhere in a template.
+
+      <details>
+        <summary>
+          More information
+        </summary>
+
+        You may import this method as needed in the views that will render blocks. Or, as a convenience, you may `import` this function automatically in all of your phoenix views by adding it to the `your_app_web.ex` definition for `view` (or `view_helpers` if you want it to be available for live views as well, example below).
+      </details>
+
+  2. In addition, to render the block editor in your forms, you'll want to pull in `ColonelKurtz.FormHelpers` too. The example below shows how to do this for all Phoenix Views in your application.
+
+      <details>
+        <summary>
+          See an example
+        </summary>
+
+        ```elixir
+        # lib/your_app_web.ex
+
+          # ...
+
+          defp view_helpers do
+            quote do
+              use Phoenix.HTML
+
+              import Phoenix.LiveView.Helpers
+              import BlogDemoWeb.LiveHelpers
+
+              # 1. import `render_blocks/1` so that it is available for all views
+              import ColonelKurtz.Renderer, only: [render_blocks: 1]
+
+              # 2. import the form helpers to make `block_editor/2` available in your form templates
+              import ColonelKurtz.FormHelpers
+
+              import Phoenix.View
+
+              import BlogDemoWeb.ErrorHelpers
+              import BlogDemoWeb.Gettext
+
+              alias BlogDemoWeb.Router.Helpers, as: Routes
+            end
+          end
+          # ...
+        ```
+      </details>
+
+  3. Render your blocks inside your view's show template:
+
+      ```elixir
+      # lib/your_app_web/templates/blocks/show.html.eex
```

Comment:
Is this supposed to be in the `templates/blocks` dir? Wouldn't this just be used anywhere?

Reply:
Ah, good catch. For the given example this would be the templates for a `Post` (or arguably used in any view that has a field w/ ck blocks in it). Willfix.